packages/election/graphql/resolvers/Candidacy.js

```diff
@@ -0,0 +1,27 @@
+const { Roles, transformUser } = require('@orbiting/backend-modules-auth')
+
+module.exports = {
+  user: (candidacy, args, { user: me }) => transformUser(candidacy.user),
+  yearOfBirth: (candidacy, args, { user: me }) => {
+    if (!Roles.userIsInRoles(me, ['admin', 'supporter', 'associate'])) {
+      return ''
```

Comment:
will make boom, because `''` is not an `int!`

Reply:
IMO no access check needed here: if one is allowed to access a candidacy object they should also be allowed to read the `yearOfBirth`